spec/rspec/core/formatters/snippet_extractor_spec.rb

```diff
@@ -31,22 +31,20 @@ module RSpec::Core::Formatters
     let(:error) do
       begin
         source
-      rescue ExpectedError => error
+      rescue => error
         error
-      ensure
-        raise 'No ExpectedError has been raised' unless error
+      else
+        raise 'No error has been raised'
       end
     end
 
-    ExpectedError = Class.new(StandardError)
```

Comment:
Because the added example expects `ArgumentError` raised from `def obj.foo(arg)`